Sources/FluentKit/Query/Builder/QueryBuilder.swift

```diff
@@ -11,6 +11,7 @@ public final class QueryBuilder<Model>
     internal var shouldForceDelete: Bool
     internal var models: [Schema.Type]
     public var eagerLoaders: [AnyEagerLoader]
+    public var history: QueryHistory
```

Comment:
Do we need to hold onto this? Can't we just reference `self.database.history` like the logger or event loop?

Reply:
Good point